src/ResponseValidator.test.ts

```diff
@@ -453,7 +453,7 @@ describe("ResponseValidator", () => {
     describe("_mergeClaims", () => {
         it("should merge claims", () => {
             // arrange
-            const c1 = { a: "apple", b: "banana" } as UserProfile;
+            const c1 = { a: "apple", b: "banana" } as unknown as UserProfile;
```

Comment:
This cast is fine for now, a more proper fix will probably be to make `_mergeClaims` generic with something like `_mergeClaims<T extends JwtClaims>(claims1: T, ...): T`. Thanks for going through all of these tests.